polyfill/lib/ecmascript.mjs

```diff
@@ -571,11 +571,8 @@ export const ES = ObjectAssign({}, ES2019, {
       const value = bag[property];
       if (value !== undefined) {
         any = any || {};
-        if (property === 'calendar') {
+        if (property === 'era') {
           // FIXME: this is terrible
```

Comment:
Still terrible? :smile:

Reply:
I still think it's terrible to special-case any property here, only slightly less so than before...

Reply:
Fair enough, will merge in that case.